networkit/cpp/linkprediction/test/LinkPredictionGTest.cpp

```diff
@@ -242,4 +245,69 @@ TEST_F(LinkPredictionGTest, testKatzRunOnOrdering) {
     }
 }
 
+TEST_F(LinkPredictionGTest, testAdamicAdarIndex) {
+    METISGraphReader graphReader;
+    Graph G = graphReader.read("input/jazz.graph");
+
+    AdamicAdarIndex AAI(G);
+    auto preds = AAI.runAll();
+    for (auto pred : preds) {
```

Comment:
It's hard to understand the type of `preds` and `pred` without knowing the type returned by `runAll`. I suggest writing the type explicitly. Also, `pred` can be `const &` since you are not changing it.